.circleci/config.yml

```diff
@@ -266,6 +274,22 @@ jobs:
           command: |
             ./scripts/publish-release.sh
 
+  build_faucet_and_genesis:
+    docker:
+      - image: circleci/golang:1.11.1
+    working_directory: /go/src/github.com/filecoin-project/go-filecoin
+    resource_class: small
+    steps:
+      -run:
+         command: |
+           go build -o ./faucet ./tools/faucet/main.go
```

Comment:
I believe that this won't build without running `go run ./build/*go deps` first.  
Maybe `build_linux` is a better place for this since we run `go deps` there already (and it's slow)

Reply:
instead I added deps from cache